remo/profiles/models.py

```diff
@@ -216,9 +216,11 @@ class UserProfile(models.Model):
     functional_areas = models.ManyToManyField(
         FunctionalArea, related_name='users_matching')
     mobilising_skills = models.ManyToManyField(
-        MobilisingSkill, related_name='users_matching_skills')
+        MobilisingSkill, related_name='users_matching_skills',
+        blank=True)
```

Comment:
I would suggest to add a default value of `default=''`.

Reply:
done :)